lib/apia/rack.rb

```diff
@@ -152,18 +152,41 @@ def triplet_for_exception(exception)
 
     class << self
 
+      # Return a plain text triplet for the given body.
+      #
+      # @param body [String]
+      # @param status [Integer]
+      # @param headers [Hash]
+      # @return [Array]
+      def plain_triplet(body, status: 200, headers: {})
+        response_triplet(body, content_type: 'text/plain', status: status, headers: headers)
+      end
+
       # Return a JSON-ready triplet for the given body.
       #
       # @param body [Hash, Array]
       # @param status [Integer]
       # @param headers [Hash]
       # @return [Array]
       def json_triplet(body, status: 200, headers: {})
-        body_as_json = body.to_json
+        response_triplet(body.to_json, content_type: 'application/json', status: status, headers: headers)
+      end
+
+      # Return a triplet for the given body.
+      #
+      # @param body [Hash, Array]
+      # @param content_type [String]
+      # @param status [Integer]
+      # @param headers [Hash]
+      # @return [Array]
+      def response_triplet(body, content_type:, status: 200, headers: {})
+        content_length = body.bytesize.to_s
```

Comment:
I believe that `bytesize` is only available on `String` objects. We should probably do a responds to check for it, and simply not set `Content-Length` if it does not respond to the method.

However, we don't directly expose a means of sending a custom response. While the `Response#plain_text_body` method does pass the `body` value through as is, I would considering giving it anything other than a `String` object as incorrect usage.

Hence, I'd say this is not a blocker for merging this PR, but something that needs to be dealt with in the future should we want to expose fully custom responses.